search content pages (#111)
Informasjonsforvaltning/docs: content/Felles datakatalog/Search/_index.md

---
title: Search
weight: 8
---

There are 3 different ways to search the data in Data.norge.no, each is based on a different technology.

### Search

> The main search service in Data.norge.no is based on ElasticSearch. This service only searches the text in a few fields, but has filters and other advanced functionality that helps users navigate.

Reviewer: ```suggestion
The main search service in Data.norge.no is based on ElasticSearch. This service searches the text in a limited selection of fields, and has filters and other advanced functionality that helps users navigate.
```

> The main search service in Data.norge.no is based on ElasticSearch. This service only searches the text in a few fields, but has filters and other advanced functionality that helps users navigate.

Author: og selv om vi ikke bruker det i portal per nå så er det mulig å definere hvilke av de 3 feltene en vil søke i, kan få inn litt info om det i hovedsida til search-service og legge til link herfra

> The main search service in Data.norge.no is based on ElasticSearch. This service only searches the text in a few fields, but has filters and other advanced functionality that helps users navigate.

Author: 220a04f

> The main search service in Data.norge.no is based on ElasticSearch. This service only searches the text in a few fields, but has filters and other advanced functionality that helps users navigate.

Author: og 138fc0e

### LLM

The LLM search service uses a 'Large Language Model' (LLM) that is built from the same data used in the main search, but will accept naturally worded queries where the other search demands exact wording of titles or keywords.

> 

Author: 138fc0e

### SPARQL

> The SPARQL search service is based on the RDF query language SPARQL, and is our most advanced and powerful search. A user will have to create queries that follows the correct syntax, but will be able to search all harvested data points, not just pre-selected fields.

Author: 138fc0e
